src/components/shadows/examples/surface-shadows.tsx

```diff
@@ -0,0 +1,53 @@
+import { Component, h } from '@stencil/core';
+
+@Component({
+    tag: 'limel-example-surface-shadows',
+    shadow: true,
+    styleUrl: 'shadow-examples.scss',
+})
+export class SurfaceShadowExample {
+    public render() {
+        return [
+            <div class="shadow-example-grid">
+                <div class="visualization">
+                    <div class="surface shadow-depth-8">
+                        <div class="label">Depth 8</div>
+                    </div>
+                    <code>--shadow-depth-8</code>
+                    <p>
+                        suitable for command bars, command dropdowns, context
+                        menus
+                    </p>
+                </div>
+                <div class="visualization">
+                    <div class="surface shadow-depth-16">
+                        <div class="label two">Depth 16</div>
+                    </div>
+                    <code>--shadow-depth-16</code>
+                    <p>
+                        suitable for teaching callouts, search results dropdown,
+                        cards, or tooltips
+                    </p>
+                </div>
+                <div class="visualization">
+                    <div class="surface shadow-depth-16">
+                        <div class="label two">Depth 64</div>
+                    </div>
+                    <code>--shadow-depth-64</code>
+                    <p>suitable for modals, dialogues, or pop overs.</p>
```

Comment:
US English in programming, so it's "dialogs"…

```suggestion
                    <p>suitable for modals, dialogs, or pop overs.</p>
```